.github/labeler.yml

```diff
@@ -50,4 +50,11 @@ feature/web3/wallet/core:
       - browser/resources/settings/brave_wallet_page/**/*
       - browser/ui/brave_wallet/**/*
       - browser/ui/webui/brave_wallet/**/*
-      - components/brave_wallet/**/*
\ No newline at end of file
+      - components/brave_wallet/**/*
+
+# add this label only if doc files have changed. This will skip CI runs
```

Comment:
`# only-docs label skips CI (including noplatform) and is for docs/md file changes`